cfme/tests/cloud_infra_common/test_tag_visibility.py

```diff
@@ -28,6 +28,7 @@ def tagged_vm(tag, provider):
     yield tag_vm
     tag_vm.appliance.server.login_admin()
     tag_vm.remove_tag(tag=tag)
+    tag_vm.delete()
```

Comment:
**QUESTION** i recall the capandu_vm being absolutely required to left running - in particualr since at least the visible code here we never create one, we only add a tag

so how does this play together?

Reply:
tag_vm is EC2Instance and delete removes it from CFME database, not entirely.
This instance is created when we set up provider but it's not removed when removing provider (actually, seems like it's true only for Openstack)